components/style/properties/helpers.mako.rs

```diff
@@ -33,6 +33,81 @@
     </%call>
 </%def>
 
+<%doc>
+    To be used in cases where we have a grammar like
+    "<thing> [ , <thing> ]*", but only support a single value
+    in servo
+</%doc>
+<%def name="gecko_autoarray_longhand(name, **kwargs)">
```

Comment:
once servo starts supporting `<thing> [ , <thing> ]*` properties we can refactor this to work in general and have a thin conditional-compile wrapper around it.